src/renderer/views/pool/PoolHistoryView.tsx

```diff
@@ -37,10 +39,6 @@ export const PoolHistory: React.FC<Props> = ({ className, poolAsset }) => {
 
   const { getExplorerTxUrl$ } = useThorchainContext()
 
-  /**
-   * TODO @asgdxteam Remove `network` param
-   * after https://github.com/xchainjs/xchainjs-lib/issues/287 is resolved
-   */
   const { network$ } = useAppContext()
```

Comment:
i suppose we can use directly `clientNetwork$` if provide it to the `AppContext` or with previous comment just use `network` if there is `getClientNetwork` inside of `network$`